### PR TITLE
Check if exist follower when request snapshot from followers

### PR DESCRIPTION
**Pull request by qian0817**
### What changes are proposed in this pull request?

Check if exist follower when request snapshot from followers.

### Why are the changes needed?

If there is only the master node, then the log file will be filled with `Failed to request snapshot info from followers No recent snapshot found from followers`  when request snapshot from followers.

### Does this PR introduce any user facing changes?

no user facing changes.

### CODE REVIEW

**Review by ggezer — APPROVED** [review]

**Comment by ggezer** [comment]
Will this now spin rather than printing? 
Should we put a sleep here?

**Comment by qian0817** [comment]
> Will this now spin rather than printing?
> Should we put a sleep here?

Now when the checkpoint threshold is exceeded, the log will be printed when a raft log is applied.

**Comment by ggezer** [comment]
I meant to ask how frequently this new `return` statement will be hit?

**Comment by qian0817** [comment]
Almost all entries will be hit this statement if there is no follower.

**Comment by ggezer** [comment]
@qian0817 I'm trying to understand  if there's a busy-loop here. If yes, then putting a sleep here will reduce CPU pressure even though count remains the same.

**Comment by qian0817** [comment]
@ggezer It's strictly not a loop, but I find that adding sleep will reduce CPU pressure. I have already added the statement.

File: core/server/common/src/main/java/alluxio/master/journal/raft/SnapshotReplicationManager.java [hunk 1]
```diff
@@ -449,6 +450,12 @@ private void requestSnapshotFromFollowers() {
                   .newBuilder()
                   .setSnapshotInfoRequest(GetSnapshotInfoRequest.getDefaultInstance())
                   .build()))));
+      // no follower in the cluster
+      if (jobs.isEmpty()) {
+        CommonUtils.sleepMs(Constants.SECOND_MS);
```
Comment (ggezer) [inline core/server/common/src/main/java/alluxio/master/journal/raft/SnapshotReplicationManager.java:455]: Let's make this a minute.

**Comment by qian0817** [comment]
solved by #13456 ,close it.

**Pull request closed without merging**